lib/templates.ts

```diff
@@ -124,23 +124,34 @@ async function _updateTemplate(
 }
 
 /**
- * Deletes a form template
+ * Deletes a form template. The template will stay in the database for 30 days in an archived state until a lambda function deletes it from the database.
  * @param formID ID of the form template
  * @returns A boolean status if operation is sucessful
  */
 async function _deleteTemplate(formID: string): Promise<FormRecord | null> {
   try {
-    const deletedTemplate = await prisma.template.delete({
+    const dateIn30Days = () => {
```

Comment:
If this function is being used once. It might as well just be a variable

Reply:
I do not think this is so much of a concern. The variable is used almost immediately after within the function. But if you wanted you could do the following 

```js
const date = new Date(Date.now() + 2592000000)
```